src/lib/crypto/kimchi_bindings/stubs/dune

```diff
@@ -13,8 +13,6 @@
   (source_tree ../../proof-systems))
  (action
   (progn
-   (run rm -rf ./target)
-   (run rm -f ./Cargo.lock)
```

Comment:
btw I had some problems without these commands. If you run `cargo build` in `src/lib/crypto/kimchi_bindings/stubs` then the `target/` folder and the `Cargo.lock` files are created, and they would interfere with the `dune build` command for some reason. I'm not sure it's still a problem since we don't list these in the `deps` field, but if we run into this problem again it's good to remember that comment